src/components/TodayNumbersSection/index.js

```diff
@@ -30,7 +30,11 @@ function TodayNumbersSection({ updateDate, title, children }) {
                   {updateDate}
                 </section>
                 <section className='w-100 h-50'>
-                  <Card bodyClassName='bg-soft-blue' href='/'>
+                  <Card
+                    bodyClassName='bg-soft-blue'
+                    className='fr-card--no-border'
```

Comment:
props `hasBorder={false}` does not work ?

Reply:
Thank you, I did not know this attr !